### PR TITLE
Remove filename from entries inside the TRANSCODE folder

### PR DESCRIPTION
**Pull request by SubJunk**

### CODE REVIEW

File: src/main/java/net/pms/dlna/DLNAResource.java [hunk 1]
```diff
@@ -1604,6 +1604,10 @@ public String getDisplayName(RendererConfiguration mediaRenderer) {
 		return getDisplayName(mediaRenderer, true);
 	}
 
+	public boolean isInsideTranscodeFolder() {
+		return parent != null && parent instanceof FileTranscodeVirtualFolder;
```
Comment (Nadahar) [inline src/main/java/net/pms/dlna/DLNAResource.java:1608]: `parent instanceof FileTranscodeVirtualFolder` also checks that it's non-null. ´null´ is never the instance of anything.

Comment (SubJunk) [reply src/main/java/net/pms/dlna/DLNAResource.java]: Thanks, I removed that check

**Comment by SubJunk** [comment]
@valib this is passing now

**Comment by SubJunk** [comment]
@valib can this be merged?

**Review by valib — APPROVED** [review]

**Pull request merged**